src/test/java/org/mule/tools/apikit/input/ConfigurationPropertiesConfigParserTest.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.tools.apikit.input;
+
+import org.jdom2.Document;
+import org.jdom2.JDOMException;
+import org.junit.Test;
+import org.mule.tools.apikit.TestUtils;
+import org.mule.tools.apikit.input.parsers.APIAutodiscoveryConfigParser;
+import org.mule.tools.apikit.input.parsers.ConfigurationPropertiesConfigParser;
+import org.mule.tools.apikit.model.APIAutodiscoveryConfig;
+import org.mule.tools.apikit.model.ConfigurationPropertiesConfig;
+
+import java.io.IOException;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+public class ConfigurationPropertiesConfigParserTest {
```

Comment:
Same comments as the previous Test class for this one. Rename and use rule for asserting the exception thrown